docs/classes/Schema.md

```diff
@@ -13,7 +13,7 @@ let schema = new Schema(Foo, {
   aBoolean: { type: 'boolean' },
   anArray: { type: 'array' }
 }, {
-  dataStructure: 'JSON'
+  dataStructure: 'JSON
```

Comment:
whoops!

Reply:
Odd. Simon just fixed that.

Reply:
Thought I merged it.

Reply:
Ah. Was fixed in the docs, which are generated, not the source.